data/remnant/remnant missions.txt

```diff
@@ -1061,138 +1061,646 @@ mission "Remnant: Learn Sign Follow Up"
+			
+			branch surveillance
+				has "remnant: told about surveillance"
+			
+			choice
+				`	"I was wondering what your research on those Remnant ship scans has brought up."`
+				`	"Could I get a copy of your research on those Remnant ship scans?"`
+					goto suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	"The ship scans look like nothing we've ever seen before. The ships seem to have a sort of... organic property to them. Here, let me show you." Ivan pulls up a schematic on his computer of the Remnant ships that you scanned. You're somewhat surprised that he could pull so much information from an outfit scan. "The ships look relatively normal for the most part, but the hull of these ships have an almost organic structure to them." He zooms in on a section of the schematic that highlights the hull structure. "Now when it comes to ships designed by say Betelgeuse or even the Navy, the hull supports are arranged in triangular or square sections. But these ships have a pseudo-hexagonal design to them, completely non-uniform. It almost looks like the cell structure of some plants.`
+			`	"Aside from the strange hull structure, these ships also seem to have a considerably amount of equipment integrated directly into the ships themselves. In-built scanners and ramscoops, for example. Stuff that didn't come up on the surface of the outfit scan but that we can tell is there. The smaller of the two ships you scanned also has some sort of in-built device that we can't make any sense of. It doesn't look like any sort of device that we've seen before, but it takes up a considerable portion of the ship's structure, so it must have some purpose. We'd need to get a closer look to figure out what it is."`
+			`	"Do you think I could get a copy of your research?" you ask.`
+				goto suspicion
+			
+			label technology
+			choice
+				`	"I was wondering what your research on Remnant technology has brought up."`
+				`	"Could I get a copy of your research on Remnant technology?"`
+					goto suspicion
+			branch cubes
+				has "Deep: Remnant Surveillance: done"
+			`	"Ah. Well, there's quite a bit to talk about." Ivan reclines his chair.`
+			`	Ivan pulls out a schematic showing the engines you brought. "The engines you brought us are very interesting. We already mentioned to you how they're rather powerful for their size, more so than even atomic engines of a similar, while not producing very much heat at all. Their major downside though seems to be the immense amount of energy they draw. The energy drain is not surpising given that they seem to be improved ion engines, but the fact that ion engines could ever become more powerful for their size than atomic engines is somethign spectacular. The thrusters also seem to have the ability to provide reverse thrust, making any ship with them rather maneuverable."`
```

Comment:
I think Ivan might be overselling the Remnant engines a bit with that first statement. In terms of space, the closest atomic thruster to the smelter is the A520 which has both more total thrust and more thrust per ton. Although for the Forge class the comparison seems to be other way around, with the A250 having both less thrust and less thrust/ton. Interesting. I guess whether or not Ivan is overselling them basically comes down to which set of engines he got.

Reply:
I'll need to rework these anyway to make mention of the slowing resistance.

Reply:
Do they need to make mention of it? The Deep doesn't have any easy access to slowing tech of any kind, so it seems unlikely that they'd be able to identify that ability within it. Perhaps simply a mention of how they have an unusual design or things that they don't understand the purpose of it.

Reply:
Well they do have the Inhibitor, so they could have shot at a ship that had the engines equipped with the Inhibitor. Although in that case we need to make sure the player also brought the Inhibitor, which they might not have.